spec/javascripts/govuk_publishing_components/analytics-ga4/gtm-page-views.spec.js

```diff
@@ -16,30 +16,30 @@ describe('Google Tag Manager page view tracking', function () {
         title: 'This here page',
         status_code: '200',
 
-        document_type: 'n/a',
-        publishing_app: 'n/a',
-        rendering_app: 'n/a',
-        schema_name: 'n/a',
-        content_id: 'n/a',
-
-        section: 'n/a',
-        taxon_slug: 'n/a',
-        taxon_id: 'n/a',
-        themes: 'n/a',
-        taxon_slugs: 'n/a',
-        taxon_ids: 'n/a',
-
-        language: 'n/a',
+        document_type: null,
```

Comment:
Might there be any mileage in storing `null` in a variable and then using the variable as the value? Just thinking if we needed to change `null` to something else in future, we'd only need to change the variable

Reply:
Good shout, have updated.